README.md

```diff
@@ -2,9 +2,13 @@
 
 Plugin to show customizable overlays for Advanced Combat Tracker.
 
-## Download
+## Download / Installation
 
-You can download release and pre-release archives from the [release page][releases].
+To install OverlayPlugin, please follow the instructions
+in the [FFXIV ACT Setup Guide](https://overlayplugin.github.io/docs/setup/).
```

Comment:
Maybe this should mention specifically using the `Get Plugins` button, as well as linking to the guide? Something like:

> To install OverlayPlugin, please use the `Get Plugins` button in ACT. For more detailed instructions, please consult the [FFXIV ACT Setup Guide](https://overlayplugin.github.io/docs/setup/).